naptan_batch_getter/naptan_get_batch_stops.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+from botocore.exceptions import ClientError
+
+TABLE_NAME = 'Naptan'
+MAX_BARCH_SIZE = 100
+REGION = 'eu-west-1'
+
+
+def lambda_handler(event, context):
+
+    # Connect to DynamoDB using boto
+    dynamo_db = boto3.resource('dynamodb', region_name=REGION)
+    stop_ids = event["stop_ids"]
+    stops_ids_list = stop_ids.split(',')
+
+    if len(stops_ids_list) > MAX_BARCH_SIZE:
+        print('Cannot support batch get item for more then 100 items')
+        raise ClientError
```

Comment:
I think the raise should also include the error message so the client knows why it failed.

Reply:
done